tests/perf/components/telemetry/deploy/telemetry-config.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+data:
+  custom_parsers.conf: |
+    [PARSER]
+        Name docker_no_time
+        Format json
+        Time_Keep Off
+        Time_Key time
+        Time_Format %Y-%m-%dT%H:%M:%S.%L
+  fluent-bit.conf: |
+    [SERVICE]
+        Daemon Off
+        Flush 1
+        Log_Level warn
+        Parsers_File parsers.conf
+        Parsers_File custom_parsers.conf
+        Parsers_File dynamic-parsers/parsers.conf
+        HTTP_Server On
+        HTTP_Listen 0.0.0.0
+        HTTP_Port 2020
+        Health_Check On
+        storage.path /data/log/flb-storage/
+
+    [INPUT]
+        Name tail
+        Path /var/log/containers/*.log
+        multiline.parser docker, cri
+        Tag kube.*
+        Mem_Buf_Limit 5MB
+        Skip_Long_Lines On
+        Refresh_Interval 10
+        DB /data/flb_kube_loki.db
+
+    [FILTER]
+        Name kubernetes
+        Match kube.*
+        Merge_Log On
+        K8S-Logging.Parser On
+        K8S-Logging.Exclude On
+        Buffer_Size 1MB
+
+    @INCLUDE dynamic/*.conf
+
+kind: ConfigMap
```

Comment:
This Fluent Bit configuration is required for the test? It is not mentioned in the README to be applied.

Reply:
@rakesh-garimella added it. What exactly is it for?

Reply:
We need this config as we need to enable `storage.path /data/log/flb-storage/` and move  `@INCLUDE dynamic/*.conf` to the bottom so that k8s filters are applied before being passed into rewrite_tag filter. We should add this to readme will do that.

Reply:
Please add some copy-pasteable command to apply this config